scene/gui/text_edit.cpp

```diff
@@ -4024,7 +4024,7 @@ void TextEdit::redo() {
 	}
 	_push_current_op();
 
-	if (undo_stack_pos == nullptr) {
+	if (!has_redo()) {
```

Comment:
Should the same change be done in `undo()`?

Reply:
Like I said, there's a bit more complexity with the function and the bit of code it's intended to replace, so I decided to lave it be.